graphsdk/src/androidTest/java/com/microsoft/graph/serializer/DateSerializerTest.java

```diff
@@ -0,0 +1,32 @@
+package com.microsoft.graph.serializer;
+
+import android.test.AndroidTestCase;
+
+import com.microsoft.graph.model.Date;
+
+public class DateSerializerTest extends AndroidTestCase {
```

Comment:
Could you create a ReocurranceRange and make sure it can be serialized and deserialized with the expected output?